app/assets/javascripts/discourse/app/components/image-uploader.js

```diff
@@ -82,9 +74,19 @@ export default Component.extend(UploadMixin, {
   },
 
   _openLightbox() {
-    next(() =>
-      $(this.element.querySelector("a.lightbox")).magnificPopup("open")
-    );
+    next(() => {
+      // get the gallery for current uploader. "lg-uid" is added by lightgallery
```

Comment:
is this a data-attribute ?

Reply:
It is, is there any additional information that you think I should add to that comment?

Reply:
mostly you should be using dataset to get data attributes now